src/components/ActionMenu/components/MenuGroup/MenuGroup.tsx

```diff
@@ -30,6 +30,10 @@ export class MenuGroup extends React.Component<MenuGroupProps, never> {
       icon,
     } = this.props;
 
+    if (!actions || (actions && actions.length === 0)) {
```

Comment:
I believe you only need to check `actions.length === 0`.

Reply:
I believe, since it's not required, someone can pass 

```
{
  title: "Test",
}
```

into `actionGroups` prop and it will still render the element, see https://codesandbox.io/embed/sleepy-austin-4kkxt

Reply:
Both `title` and `actions` are required in Typescript and raise an error,

https://github.com/Shopify/polaris-react/blob/5429427022bf09eba8c06007005e6711bf165b3a/src/types.ts#L188-L192

But there doesn't appear to be any errors/warnings in JS. Probably best to leave in the extra checks then 👍

Reply:
> I believe, since it's not required, someone can pass

As Laura mentioned above it is a required prop, and using the API without `actions` isn't support, and we usually rely on typescript to catch these errors.

Given that, `if ( actions.length === 0)) return null;` or `if (!actions.length)) return null;` should suffice for a check. With that being said, if you were looking to use the API without actions, we could look into that. What's your use case?

Reply:
No particular use case, just wanted to add a safety net for developers that are using Polaris without TS that could potentially add an `actionGroups` item without actions thinking it's a secondary action 🤷‍♂ but since it seems like an overkill, updated to a length check 👍 Thanks for your input!